pykafka/balancedconsumer.py

```diff
@@ -398,6 +400,10 @@ def _set_watches(self):
             self._zookeeper, self._consumer_id_path,
             self._consumers_changed
         )
+
+        self._zookeeper.add_listener(self._zookeeper_state_changed)
+        self._zookeeper_connected = self._zookeeper.state == KazooState.CONNECTED
```

Comment:
Is it safe/necessary to write to `_zookeeper.state` here? Won't the client manage that correctly by itself?